admin/controllers/questions.js

```diff
@@ -35,6 +35,23 @@ const getQuestions = async (req, res) => {
     return apiResponse.unauthorised(res)
   }
 
+  try {
```

Comment:
are we not adding any test for this case?

Reply:
In short - No.  Had a discussion about this earlier in the week and created a tech debt card for it.  The entire controller tests need refactoring, and there will be further work on the controller itself to rehouse these key parts of logic into one service.

its become very fragmented, with various service calls tripping over one another.   Its evolved into that over time, and just needs a good design and refactor.

Reply:
Alright